changelog.txt

```diff
@@ -30,7 +31,8 @@ OPENSEADRAGON CHANGELOG
 * Fixed issue causing HTML pages to jump unwantedly to the reference strip upon loading (#872)
 * Added addOnceHandler method to EventSource (#887)
 * Added TiledImage.fitBounds method (#888)
-* Added scaledWidth and scaleHeight options to Rect overlays to allow to scale in only one dimension.
+* Overlays can now be scaled in only one dimension by providing a point location and either width or height (#896)
```

Comment:
Let's change "only one" to "a single" ... the current wording at first made me think we were removing the ability to scale in two dimensions. 

Thanks for doing the changelog here!